deeppavlov/core/agent/agent.py

```diff
@@ -47,13 +48,15 @@ class Agent(Component, metaclass=ABCMeta):
             handles automatic state management for skill. All skills are
             wrapped to SkillsWrapper automatically during agent initialisation.
             We highly recommend to use wrapped skills for skills inference.
+        dialog_logger: DeepPavlov dialog logging facility.
     """
     def __init__(self, skills: List[Skill]) -> None:
         self.skills: List[Skill] = skills
         self.history: Dict = defaultdict(list)
         self.states: Dict = defaultdict(lambda: [None] * len(self.skills))
         self.wrapped_skills: List[SkillWrapper] = \
             [SkillWrapper(skill, skill_id, self) for skill_id, skill in enumerate(self.skills)]
+        self.dialog_logger: DialogLogger = DialogLogger()
```

Comment:
maybe allow to set agent_name in agent's constructor? Something like
```python3
def __init__(self, skills: List[Skill], *, log_name: Optional[str] = None) -> None:
  ...
  self.dialog_logger: DialogLogger = DialogLogger(agent_name=log_name)
```

Reply:
It will be implemented in other PR with opt config scheme.